pkg/dao/dao.go

```diff
@@ -191,6 +197,18 @@ func (d *Dao) BatchGetSpecs(dir string) ([]*apb.Spec, error) {
 	return specs, nil
 }
 
+// BatchDeleteSpecs - set specs based on SpecManifest in the kvp API.
+func (d *Dao) BatchDeleteSpecs(specs []*apb.Spec) error {
+	// TODO: Is there no batch insert in the etcd api?
```

Comment:
I remember searching for this and coming up with nothing. Would love to hear anything to the contrary.